oat-sa/extension-tao-itemqti-pci
# Fix/vulnerabilities

### Related to: https://oat-sa.atlassian.net/browse/ADF-1357

### Summary

Fix a vulnerability brought by `mathml-to-latex`.

### Details

The npm link was targeting the old version for the library.

Related issues:
- https://github.com/oat-sa/extension-tao-itemqti-pci/security/dependabot/1
- https://github.com/oat-sa/extension-tao-itemqti-pci/security/dependabot/3

### How to test
- install the PCI
- check that the math rendering works well in the prompt (mathml expressions)
